### PR TITLE
Lecture 12 B

### PR DESCRIPTION
**Pull request by ashwin-97**

### CODE REVIEW

**Review by lagrassa — CHANGES_REQUESTED** [review]
### Review
1. Reviews PAC, Adaboost and MAB characteristics.
1. In general figures should be recreated. Here, this figure isn't referenced in the notes. For future scribe notes, figures should be referenced in the text so it's clearer what the purpose of the figure is.
1. Includes example MAB problems
1. Review could be improved by focusing on aspects of MAB problems that need to be known for this lecture. Ex) mathematical formulation/notation would be helpful.

### Summary
1. Figure 2 is used well and described in the text, but is copied from the slides wh
ich. Please re-create the figures and resubmit (see Syllabus for details on what can be taken directly from lecture slides and other sources)
1. In 2.2.1 good breaking down what Hoeffding's inequality means
1. Since this is the first time we use an inequality for a potential function, it wo
uld be good to briefly describe how this is used differently from a single value. At
 the very lease, please include the first step in "Combining Lower and Upper Bounds"
1. Please define $T_k$. I did not see it in the notes.
1. After Theorem 5, I don't understand your explanation for UCB starting exploitation earlier 
than EXP3 since there isn't really a defined time when EXP3 starts ``exploiting'' and stops ``exploring''
can you please elaborate on this more in the revision?

### Grader notes
1. Figure 2, 3 taken directly from slides (so needs to be redone)
1. Includes 1 page with a proof for Hoeffdings inequality as additional information

**Comment by ashwin-97** [comment]
All the changes are done as requested

**Comment by lagrassa** [comment]
Nice improvements.
